docs/integrations/editors.md

```diff
@@ -78,36 +78,51 @@ Options include the following:
 
 ## Wing IDE
 
-Wing supports black via the OS Commands tool, as explained in the Wing documentation on
-[pep8 formatting](https://wingware.com/doc/edit/pep8). The detailed procedure is:
+Wing IDE supports `black` via the **Preference Setting** for system wide settings and
+**Project Properties** for per-project or workspaces specific setting, as explained in
+the Wing documentation on
+[Auto-Reformatting](https://wingware.com/doc/edit/auto-reformatting). The detailed
+procedure is:
 
-1. Install `black`.
+### Prerequistes
 
-   ```console
-   $ pip install black
-   ```
+- Wing IDE version 8.0+
 
-1. Make sure it runs from the command line, e.g.
+- Install `black` in virtual environment or project.
```

Comment:
```suggestion
- Install `black`.
```
I'd prefer not mentioning virtual environments as people may be a bit confused by what they are, but if you think it's better then feel free to keep it this way. Same thing with 'project', unless Wing IDE supports specifying a dependency for a project I feel like it's confusing.

Reply:
Maybe, you are correct. :smiley: In Wing ide docs they just say `pip install black`, I try to be more specific but also too fast to assume how user handle their environment. Thank you for review :+1: